ert_gui/tools/file/file_dialog.py

```diff
@@ -1,4 +1,5 @@
-from qtpy.QtCore import QThread, Slot, Qt
+from qtpy.QtCore import QThread, Slot, Qt, QSize
+from math import floor
```

Comment:
For style reasons, the `math` import should be outside (ahead) of the `qtpy` imports. I fancy `isort`, which would fix this (but it might fail in case of circular imports so we cannot auto-apply it to all files)

Reply:
I think this is because PEP8 style conventions:

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.
> 

https://peps.python.org/pep-0008/#imports

Reply:
thank you! done.